test/test_collection.py

```diff
@@ -876,7 +876,7 @@ def test_bad_dbref(self):
         id_only = {'ref': {'$id': ObjectId()}}
         self.assertRaises(InvalidDocument, self.db.test.insert_one, ref_only)
         self.assertRaises(InvalidDocument, self.db.test.insert_one, id_only)
-
+x
```

Comment:
Looks like a typo? Let's remove it.

Reply:
oops not sure how that got there, removed!